make bdr ddl replicate GA en (#18568)
pingcap/docs: ticdc/ticdc-bidirectional-replication.md

To solve the problem of replicable DDLs and non-replicable DDLs, TiDB introduces the following BDR roles:

> - `PRIMARY`: You can execute replicable DDLs, but not non-replicable DDLs. Replicable DDLs will be sent to the downstream by TiCDC.

Reviewer: ```suggestion
- `PRIMARY`: You can execute replicable DDLs, but not non-replicable DDLs. Replicable DDLs executed in a PRIMARY cluster will be replicated to the downstream by TiCDC.
```

- `SECONDARY`: You cannot execute replicable DDLs or non-replicable DDLs. However, DDLs executed in a PRIMARY cluster can be replicated to a SECONDARY cluster by TiCDC.

> When not setting the BDR role, you can execute any DDL. However, the changefeed in BDR mode will not replicate any DDL on that cluster.

Reviewer: ```suggestion
When no BDR role is set, you can execute any DDL. However, the changefeed in BDR mode does not replicate any DDL on that cluster.
```

> In general, in BDR mode, TiCDC will only replicate replicable DDL on the Primary cluster to the downstream.

Reviewer 2: ```suggestion
In general, in BDR mode, TiCDC will only replicate replicable DDL on the PRIMARY cluster to the downstream.
```

> In general, in BDR mode, TiCDC will only replicate replicable DDL on the PRIMARY cluster to the downstream.

Reviewer: ```suggestion
In short, in BDR mode, TiCDC only replicates replicable DDLs in the PRIMARY cluster to the downstream.
```
